business/openshift_oauth.go

```diff
@@ -0,0 +1,205 @@
+package business
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+
+	kube "k8s.io/client-go/kubernetes"
```

Comment:
again, the imports are not right. Its a nitpick I know - but for consistency sake, put the kiali imports in the third group (the last).. the third party imports are in the middle, second group.